dkarlovi/symfony-flex-docker/0.1/bin/dphp

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
What's the point of this file? Why not simply using a "tools" service with docker-compose for such task?